packages/util/package.json

```diff
@@ -61,8 +61,18 @@
       "url": "https://github.com/asinyagin"
     }
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./provider": {
+      "import": "./dist/esm/provider.js",
+      "require": "./dist/cjs/provider.js"
+    }
```

Comment:
I'm good with it too, though would you rather me figure out a better way to rewrite the tests?  I get the impression that if we're having to explicitly define this provider export just to satisfy a test, it's my test and not our export configuration that should be fixed.

Reply:
It would be great if we can fix the test, I went really deep down this rabbit hole though and tried a lot. But maybe you've got another approach I didn't think of. Not sure if it's worth another 3 hours of work time though.